src/main/git-elegant-configure

```diff
@@ -30,7 +30,7 @@ _configure() {
             read answer
             answer=${answer:-$default}
             if [ -n "${answer}" ]; then
-                git config $MODE $(eval "echo -n \$${f}_key") $answer
+                git config $MODE $(eval "echo -n \$${f}_key") "$answer"
```

Comment:
Please add a unit test.

Reply:
@extsoft But we actually mock `git config $MODE $(eval "echo -n \$${f}_key") "$answer"` command during unit testing. So there will be no difference whether I mock it like:
`fake-pass git "config --global user.name UserName"`
or
`fake-pass git "config --global user.name User Name"`

Reply:
@AntonKarabaza Then let's wait for #89.